freeipa.spec.in

```diff
@@ -101,9 +101,13 @@
 
 # fix for segfault in python3-ldap, https://pagure.io/freeipa/issue/7324
 %global python_ldap_version 3.1.0-1
-# 1.4.3 moved nsslapd-db-locks to cn=bdb sub-entry
-# https://pagure.io/freeipa/issue/8515
-%global ds_version 1.4.3
+# https://github.com/389ds/389-ds-base/issues/4526
+# sync_repl issue
+%if 0%{?fedora} >= 33
+%global ds_version 1.4.4.12-1
+%else
+%global ds_version 1.4.3.18-1
```

Comment:
#4526 is fixed by 1.4.3.19-1 on f32:
https://koji.fedoraproject.org/koji/buildinfo?buildID=1708150

Reply:
Thanks for noticing! Updated.